lightningd/bitcoind.c

```diff
@@ -655,6 +663,25 @@ void bitcoind_getblockhash_(struct bitcoind *bitcoind,
 			  "getblockhash", str, NULL);
 }
 
+void bitcoind_gettxout(struct bitcoind *bitcoind,
+		       const struct bitcoin_txid *txid, const u32 outnum,
+		       void (*cb)(struct bitcoind *bitcoind,
+				  const struct bitcoin_tx_output *txout,
+				  void *arg),
+		       void *arg)
+{
+	tal_t *tmpctx = tal_tmpctx(bitcoind);
+
+
+	start_bitcoin_cli(bitcoind, NULL,
+			  process_gettxout, true, cb, arg,
+			  "gettxout",
+			  take(type_to_string(tmpctx, struct bitcoin_txid, txid)),
+			  take(tal_fmt(tmpctx, "%u", outnum)),
+			  NULL);
+	tal_free(tmpctx);
+}
```

Comment:
You don't actually use tmpctx here: you take() both the pointers (and yes, start_bitcoin_cli does TAKE for those args correctly, I checked).